starter-code/css/main.css

```diff
@@ -6,3 +7,164 @@ Green: #00B172
 White: #FFF
 
 */
+
+body {
+  font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;*/
+}
+
+.navbar{
+  top:0px;
+  width:98.6%;
+  background:White;
+  display:flex;
+  justify-content: flex-start;
+  position:fixed;
+  border-bottom: 1px solid black;
+  z-index:1;
+}
+
+.iconspotify{
+  display:block;
+  padding-top: 10px;
+  padding-bottom: 20px;
+  padding-left: 30px;
+}
+
+.iconspotify:hover {
+ animation-name : bounce;
+ position: relative;
+ animation-iteration-count : infinite;
+ animation-duration: 2s;
+}
+
+@keyframes bounce {
+  0% {
+    margin-top : 50px;
```

Comment:
`margin-top`will push the surrounding. Because `.iconspotify:hover` has `position: relative` you can use `top` to nudge the element instead of using `margin-top`. Replace all the `margin-top` in your animation your `top` and see what happens!